ci-operator/jobs/stackrox/stackrox/stackrox-stackrox-master-presubmits.yaml

```
@@ -304,6 +304,57 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )gke-postgres-scale-tests,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

```suggestion
    always_run: false
```

as @gavin-stackrox suggested.
Compare with `pull-ci-stackrox-stackrox-master-gke-ui-e2e-tests`.